src/h5web/vis-packs/core/heatmap/HeatmapMesh.tsx

```diff
@@ -155,7 +153,7 @@ function HeatmapMesh(props: Props) {
   };
 
   return (
-    <VisMesh>
+    <VisMesh scale={[1, ordinateConfig.flip ? -1 : 1, 1]}>
```

Comment:
Working on this, I found a way easier method to flip the Y axis that I use now in `HeatmapMesh` as well.